data/Templates/Json/SimplePatient.liquid

```diff
@@ -0,0 +1,54 @@
+{
+    "resourceType": "Patient",
+    "id": "{{ msg.PatientId | to_json_string | generate_uuid }}",
```

Comment:
I'm just curious the key `msg` is used here the same as `C-CDA`, so will we use `msg` for all the datatypes except `HL7v2`?

Reply:
Yes, I think we can use `msg` for all data types except `hl7v2` (due to legacy compatibility) to keep consistency.